lib/src/equality.dart

```diff
@@ -32,34 +32,44 @@ abstract class Equality<E> {
 
 typedef F _GetKey<E, F>(E object);
 
-/// Equality of objects using a canonical value of the object itself.
+/// Equality of objects based on derived values.
 ///
 /// For example, given the class:
+///     ```
 ///     abstract class Employee {
 ///       int get employmentId;
 ///     }
+///     ```
```

Comment:
If you're using triple-tics, you don't need to indent. Also, add `dart` after the first set so that we get syntax highlighting:

    ```dart
    code
    ````